packages/discovery-react-components/src/components/ResultsPagination/__tests__/ResultsPagination.test.tsx

```diff
@@ -116,62 +116,6 @@ describe('ResultsPaginationComponent', () => {
         pageSize: 20
       });
     });
-
-    test('will add pageSize as a pageSize selection if it is not already included', async () => {
-      const { getByText } = await setup(
-        { pageSize: 25, pageSizes: [10, 20, 30, 40, 50] },
-        {
-          searchResponseStore: {
-            ...searchResponseStoreDefaults,
-            parameters: { projectId: '', count: 25 }
-          }
-        }
-      );
-
-      expect(getByText('25')).toBeInTheDocument();
-    });
-  });
-
-  describe('when there are component settings available', () => {
-    describe('and there are no display parameters passed on ResultsPagination', () => {
-      test('will update the count search param', async () => {
```

Comment:
Why was this and following test removed?

Reply:
These are both for testing whether adjusting the `results_per_page` settings properly update the count value on the API calls. With this PR, we will no longer use `results_per_page` at all, in favor of using the `count` values from the shim, so these tests aren't applicable anymore.